src/controllers/url-controller.ts

```diff
@@ -0,0 +1,152 @@
+import { Request, Response } from 'express';
+import prisma from '../libs/prisma-client';
+
+import { convertToHyphenated, generateShortUrl } from '../utils/short-url-generator';
+import logger from '../logs/logger';
+
+const successMsg = 'Url successfully fetched';
+const notFoundMsg = 'No Url found!';
+
+//  CREATE A SHORTEN URL FROM A LONG URL
+export const createUrl = async (req: Request, res: Response) => {
+  const { url, customName } = req.body;
+
+  if (!url) return res.status(400).send({ message: 'Url is required!' });
+  try {
+    const existingUrl = await prisma.url.findFirst({
+      where: {
+        originalUrl: url,
+      },
+    });
+    if (existingUrl) {
+      return res.status(400).send({ message: 'Url already exists!' });
```

Comment:
- Create a constant file where you store all the `HTTP_STATUS` and import it from there. You shouldn't be throwing numbers around because they make the code less readable.
- Maybe make this a 409 status code. Check out what it means [here](https://www.webfx.com/web-development/glossary/http-status-codes/)

Reply:
done